tests/test_configs.py

```diff
@@ -37,6 +38,7 @@ def test_default():
             "ymldb": pathlib.Path(""),
             "grids": pathlib.Path(""),
             "operator_cards": pathlib.Path("my/ope/cards/"),
+            "operator_card_template_name": "_template",
```

Comment:
```suggestion
            "operator_card_template_name": "_template.yaml",
```
let's make it correct ...